package.json

```diff
@@ -3,9 +3,11 @@
   "version": "2.0.1-0",
   "description": "Security Context",
   "main": "js",
+  "module": "./dist/module.js",
   "files": [
     "contexts",
-    "js"
+    "dist/module.js",
+    "js/index.js"
```

Comment:
Does `constants.js` need to be in this list, or should the entire `js` folder be included?

Reply:
Yes, missing file. Fixed. Just added the one file since the remaining browser.js is just used to build the output.